site/content/docs/5.2/components/scrollspy.md

```diff
@@ -377,9 +377,9 @@ const scrollSpy = new bootstrap.ScrollSpy(document.body, {
 {{< bs-table "table" >}}
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
-| `rootMargin` | string | `0px 0px -40%` | Intersection Observer [rootMargin](https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/rootMargin) valid units, when calculating scroll position. |
+| `rootMargin` | string | `0px 0px -25%` | Intersection Observer [rootMargin](https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/rootMargin) valid units, when calculating scroll position. |
 | `smoothScroll` | boolean | `false` | Enables smooth scrolling when a user clicks on a link that refers to ScrollSpy observables. |
-| `target` | string \| jQuery object \| DOM element |  | Specifies element to apply Scrollspy plugin. |
+| `target` | DOM element | `null` | Specifies element to apply Scrollspy plugin. |
```

Comment:
it resolves strings to elements too (could use `selector`) :wink:

Reply:
Ah yep, you're right, I just took the js description, maybe we should update the js then in order to receive `string | element` (actually it's only `element` in https://github.com/twbs/bootstrap/blob/main/js/src/scrollspy.js#L51) ?

Reply:
the js has to stay as it was. It is resolved before the typeChecking

Reply:
Oh ok, I'll revert those, sorry for my misunderstanding!